client/my-sites/plans-features-main/index.jsx

```diff
@@ -379,6 +388,44 @@ export class PlansFeaturesMain extends Component {
 		return false;
 	}
 
+	renderProductsSelector() {
+		if ( ! isEnabled( 'plans/jetpack-backup' ) ) {
+			return null;
+		}
+
+		const { intervalType, siteId, translate } = this.props;
+		const products = [
+			{
+				title: 'Jetpack Backup',
```

Comment:
Seems like we're translating some strings, and holding off translating others. Should we be consistent?